fe/fe-core/src/main/java/com/starrocks/connector/iceberg/io/IOUtil.java

```diff
@@ -95,6 +97,10 @@ public static OutputFile getOutputFile(Path path) {
         return HadoopOutputFile.fromPath(path, new Configuration());
     }
 
+    public static InputFile getInputFile(Path path) {
+        return HadoopInputFile.fromPath(path, new Configuration());
```

Comment:
I think configuration should be static field. Because It takes time to create for each call

Reply:
ok, will do it.